app/code/community/Bolt/Boltpay/Helper/ConfigTrait.php

```diff
@@ -120,6 +120,19 @@ public function getPaymentBoltpayConfig($configPath, $checkoutType)
         return ($checkoutType === Bolt_Boltpay_Block_Checkout_Boltpay::CHECKOUT_TYPE_ADMIN) && !Mage::getStoreConfig('payment/boltpay/use_javascript_in_admin') ? '' : $configValue;
     }
 
+    /**
+     * @return array
+     */
+    public function getAllowedButtonByCustomRoutes()
```

Comment:
unit test please - test for empty, data w/o comma, and data /w command would be good.

Reply:
added

Reply:
I don't see any test. Did you forget to include it in PR?